packages/core/src/lib/markdown-it/patches/custom-component/htmlBlockRule.js

```diff
@@ -0,0 +1,147 @@
+const blockNames = require('markdown-it/lib/common/html_blocks');
+const { HTML_OPEN_CLOSE_TAG_RE } = require('./htmlRe');
+const inlineTags = require('./inlineTags');
+const vueReservedTags = require('./vueReservedTags');
+
```

Comment:
let's add the modified source (vuepress) to these files as well so we other devs can refer back easily

Reply:
Should I add the source link to every file? Or is it fine if we just add it to the "main" file, which is `customComponentPlugin`?

Reply:
sorry, missed the one in `index.js`, that should be fine.

The link (https://github.com/vuepress/vuepress-next/tree/main/packages/%40vuepress/markdown/src/plugins/customComponentPlugins) is broken though.

Reply:
> The link (https://github.com/vuepress/vuepress-next/tree/main/packages/%40vuepress/markdown/src/plugins/customComponentPlugins) is broken though.

Ah, there's an additional 's' at the end. Thanks for the heads up!